src/components/LandingPageBlocks/HeroBlock/HeroBlock.js

```diff
@@ -22,16 +23,16 @@ const HeroBlock = (props) => {
 			<div className="grid_wrapper">
 				<div className="section_title">
 					<h1>
-						Let our pre-vetted developers <br></br> build a{" "}
+						Let our pre-vetted developers <br className="title-animation-linebreak"></br> build a{" "}
 						<span id="code_block-animation-title"></span>
-						<br></br>for you
+						<br className="title-animation-linebreak"></br>{" "}for you
```

Comment:
Let's remove this </br> and let the animation move "for you". I actually like how it looks :)